README.md

```diff
@@ -179,8 +206,6 @@ And undeploy it with:
 make undeploy
 ```
 
-That's it! Try to visit http://localhost:8000/. :small_airplane:

```

Comment:
Why move this ?

Reply:
I think it is possible to visit http://localhost:8000/ before deployment,  so I moved it behind docker-compose up

Reply:
Ok, but description about k8s will lack this (: